fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -1034,21 +1034,20 @@ private boolean isStatisticsOrAnalyzer(StatementBase statement, ConnectContext c
     }
 
     private boolean supportedByNewPlanner(StatementBase statement, ConnectContext context) {
-        return statement instanceof QueryStmt
-                || statement instanceof InsertStmt
+        return statement instanceof AlterViewStmt
```

Comment:
could we add a method to StatementBase class? then we needn't a lot of ||.

Reply:
I prefer the obvious way, which makes it very easy to understand which statements we support and which ones we don't. If this code seems weird here, can I move it into a function and put it under the sql folder, is that ok?